networkit/cpp/graph/test/GraphGTest.cpp

```diff
@@ -132,6 +132,12 @@ void GraphGTest::SetUp() {
 
 /** CONSTRUCTORS **/
 
+TEST_F(GraphGTest, testDefaultConstructorIndexed) {
```

Comment:
It seems that you have to use parametric tests:
```suggestion
TEST_P(GraphGTest, testDefaultConstructorIndexed) {
```

Reply:
Yeah, I am not sure why though. We are mixing TEST_F and TEST_P also in other test files (same syntax). Trying to find a way around it, otherwise the same test is repeated 4x.